lib/direct_home_api/model/user.ex

```diff
@@ -55,20 +56,19 @@ defmodule DirectHomeApi.Model.User do
         :phone,
         :email,
         :photo,
-        :document,
-        :document_type,
         :password,
         :type
       ])
-      |> unique_constraint([:email])
-      |> unique_constraint([:document])
+      |> unique_constraint([:email])  
+      |> validate_format(:email, ~r/@/)    
       |> unsafe_validate_unique([:email], Repo)
-      |> unsafe_validate_unique([:document], Repo)
+      |> validate_length(:phone, min: 10)
+      |> validate_length(:password, min: 10)
```

Comment:
I think 8 is a good length to the password

Reply:
listo

Reply:
The change is ready